_collection-store/set.md

```diff
@@ -3,4 +3,4 @@ header: set
 example: CollectionStore.set(collection, params)
 ---
 
-Adds the collection to the [cache](#cache).  The params for the collection are required to use as part of a unique identifier for the collection.
+Adds the collection to the [cache](#cache).  The params for the collection are used as part of a unique identifier for the collection.
```

Comment:
can you link params to http://rendrjs.github.io/collection/#params too? :dancer: